pkg/ipam/ipam_linux.go

```diff
@@ -103,6 +103,20 @@ func ConfigureIface(ifName string, res *current.Result) error {
 		ip.SettleAddresses(ifName, 10)
 	}
 
+	if v4gw != nil {
+        	route := netlink.Route{LinkIndex: link.Attrs().Index, Gw:v4gw, Flags: 0x10}
```

Comment:
Looks like there's still some whitespace issues here?

Reply:
@syelburgi also, I think you might need to add the `Dst` option but set to 0.0.0.0 or ::0, right?

Reply:
Dst should be used, it would be 0.0.0.0/0 in this case. I don't think the flags are needed here as well?